Content.Server/Administration/BwoinkSystem.cs

```diff
@@ -227,11 +246,9 @@ protected override void OnBwoinkTextMessage(BwoinkTextMessage message, EntitySes
             }
         }
 
-        private string GenerateAHelpMessage(string username, string message, bool admin, bool noReceiver)
+        private string GenerateAHelpMessage(string username, string message, bool admin)
         {
             var stringbuilder = new StringBuilder();
-            if (noReceiver)
-                stringbuilder.Append(":sos:");
```

Comment:
why remove this?

Reply:
Because the `GenerateAHelpMessage` was moved to `ProcessQueue`, which only knows its last bookmark in the history, which doesn't give it a way to tell if it was received at the time of delivery.

Reply:
this is quite integral to the ahelp relay, since it lets admins know the message was sent to noone. you'll have to figure out a way around that then

Reply:
I a future commit will likely introduce a `Status` field for things like `Delivered`/`Read`, and I guess one value could be `Bounced` if no one was available.

Reply:
Alternatively, we could keep the list of recipients (and conversational role) with the history entry.

This could be a bit better in the event that the system expands to direct certain messages to certain admins. For instance if an admin is handling a situation with a player, other admins wouldn't get pinged for every message in their conversation. Could potentially also help if we need a mute function.

Reply:
Went with the `Status` approach. [See also.](https://github.com/space-wizards/space-station-14/pull/6004/commits/695c2ff94c5cf028bec2d95dec4e84a82bb3e9b8#diff-51cf5823eaa31f962a1936ec9020e843e0a9631ffd627f319d5d5f5ec41695d1R32-R38)